src/UI/controllers/OverlayUIController.js

```diff
@@ -75,15 +75,19 @@ export class OverlayUIController extends UIController {
         })
         this._mutObs.observe(document.body, {childList: true, subtree: true})
 
+        const position = getPosition()
+
         let delay = 0
-        if (!click && !this.elementIsInViewport(getPosition())) {
+        if (!click && !this.elementIsInViewport(position)) {
             input.scrollIntoView(true)
             delay = 500
         }
+        this.#state = 'parentShown'
```

Comment:
By setting the `parentShown` state right away we avoid multiple fires while the execution is suspended by the timeout.